subsys/secure_storage/Kconfig

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 Nordic Semiconductor
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Secure storage subsystem integration into NCS
+
+# Add the HUK library as the default AEAD key provider when supported.
+# It provides the highest level of security.
+choice SECURE_STORAGE_ITS_TRANSFORM_AEAD_KEY_PROVIDER
+	default SECURE_STORAGE_ITS_TRANSFORM_AEAD_KEY_PROVIDER_HUK_LIBRARY
+
+config SECURE_STORAGE_ITS_TRANSFORM_AEAD_KEY_PROVIDER_HUK_LIBRARY
+	bool "HUK library-derived keys"
```

```suggestion
	bool "Keys derived from HUK library"
```
Or `using HUK library`?